daemon/stats.go

```diff
@@ -58,14 +59,64 @@ func (daemon *Daemon) ContainerStats(name string, config *ContainerStatsConfig)
 				return nil
 			}
 
-			s := getStat(v)
+			statsJSON := getStatJSON(v)
+			if version.LessThan("1.21") {
```

Comment:
Put API version in `ContainerStatsConfig` is a good idea, but I prefer to use version instead of boolean, WDYT